compiler/rustc_lint/src/lints.rs

```diff
@@ -2977,6 +2977,10 @@ pub(crate) enum DeprecatedWhereClauseLocationSugg {
     },
 }
 
+#[derive(LintDiagnostic)]
+#[diag(lint_missing_crt_static)]
```

Comment:
You need `#[help]` here and call it `.help` in the fluent file.

Reply:
Thanks, that did the trick :)